app/src/main/res/layout/fragment_file_view.xml

```diff
@@ -36,18 +36,56 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             android:background="@color/black"
-            android:visibility="@{viewModel.isPDFViewVisible ? View.VISIBLE : View.GONE}" />
+            android:visibility="@{viewModel.isPDF ? View.VISIBLE : View.GONE}" />
 
         <androidx.core.widget.ContentLoadingProgressBar
             android:id="@+id/progressBar"
             style="@style/Widget.AppCompat.ProgressBar"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
+            android:layout_marginBottom="68dp"
             android:indeterminate="true"
             android:visibility="@{viewModel.isBusy ? View.VISIBLE : View.GONE}"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/fileViewError"
```

Comment:
usually its good practice to put the initials of the view type in the id. For example here it would be tvFileViewError

Similar to how the button view is defined below